hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfigHolder.java

```diff
@@ -85,6 +85,10 @@ public class HiveSyncConfigHolder {
       .defaultValue("false")
```

Comment:
@xuzifu666  Kindly remind,  there is a parameter `hoodie.datasource.hive_sync.skip_ro_suffix` when set it to true, will register table without ro suffix as ro table